inc/namespace.php

```diff
@@ -227,3 +196,20 @@ function robots_txt( string $output ) : string {
 
 	return $output;
 }
+
+/**
+ * Add the Yoast SEO sitemap index to the robots.txt file.
+ *
+ * @param string $output The original robots.txt content.
+ * @param bool $public Whether the site is public.
+ *
+ * @return string The filtered robots.txt content.
+ */
+function add_sitemap_index_to_robots( string $output, bool $public ) : string {
+	if ( $public ) {
+		$site_url = parse_url( site_url() );
```

Comment:
What's the reason for using `parse_url()` and building the URL manually here?

Do we have any concerns regarding multisite? e.g. is there a single sitemap for all sites or does each site have its own?

Reply:
Honestly was just looking at the example in the [developer docs](https://developer.wordpress.org/reference/hooks/robots_txt/#comment-4689) which seemed a decent standard to go by, but this could probably be simplified to just using `site_url`.

As far as multisite -- each site generates their own unique robots.txt and sitemap_index.xml files, there's no single sitemap for the whole network (though I suppose you could add sitemaps from other sites to the sitemap_index.xml of the main site if you really wanted to).